reference/config_files/profiles_default.rst

```diff
@@ -0,0 +1,484 @@
+relative path:
+
+.. code-block:: bash
+
+    $ conan install . -pr /abs/path/to/myprofile   # abs path
+    $ conan install . -pr ./relpath/to/myprofile   # resolved to current dir
+    $ conan install . -pr ../relpath/to/myprofile  # resolved to relative dir
+    $ conan install . -pr myprofile  # resolved to [CONAN_HOME]/profiles/myprofile
+
+Listing existing profiles in the *profiles* folder can be done like this:
+
+.. code-block:: bash
+
+    $ conan profile list
+    Profiles found in the cache:
+    default
+    myprofile1
+    myprofile2
+    ...
+
+You can also show profile's content:
+
+.. code-block:: bash
+
+    $ conan profile show
+    Host profile:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+
+    Build profile:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+
+
+.. seealso::
+
+    - Manage your profiles and share them using :ref:`reference_commands_conan_config_install`.
+    - Check the command :ref:`conan profile <reference_commands_profile>`.
+
+
+Profile rendering
+-----------------
+
+The profiles are rendered as **jinja2** templates by default. When Conan loads a profile, it immediately parses and
+renders the template, which must result in a standard text profile.
+
+Some of the capabilities of the profile templates are:
+
+- Using the platform information, like obtaining the current OS is possible because the
+  Python ``platform`` module is added to the render context:
+
+  .. code:: jinja
+
+     [settings]
+     os = {{ {"Darwin": "Macos"}.get(platform.system(), platform.system()) }}
+
+- Reading environment variables can be done because the Python ``os`` module is added
+  to the render context.:
+
+  .. code:: jinja
+
+     [settings]
+     build_type = {{ os.getenv("MY_BUILD_TYPE") }}
+
+- Defining your own variables and using them in the profile:
+
+  .. code:: jinja
+
+     {% set os = "FreeBSD" %}
+     {% set clang = "my/path/to/clang" %}
+
+     [settings]
+     os = {{ os }}
+
+     [conf]
+     tools.build:compiler_executables={'c': '{{ clang }}', 'cpp': '{{ clang + '++' }}' }
+
+
+- Joining and defining paths, including referencing the current profile directory. For
+  example, defining a toolchain which file is located besides the profile can be done.
+  Besides the ``os`` Python module, the variable ``profile_dir`` pointing to the current profile
+  folder is added to the context.
+
+  .. code:: jinja
+
+       [conf]
+       tools.cmake.cmaketoolchain:toolchain_file = {{ os.path.join(profile_dir, "toolchain.cmake") }}
+
+- Including or importing other files from ``profiles`` folder:
+
+  .. code-block:: jinja
+     :caption: profile_vars.jinja
```

Comment:
No need to use the .jinja extension, which is ugly for Conan profiles

Reply:
Sorry, I thought I removed all of them